libdnf.spec

```diff
@@ -47,8 +48,9 @@ BuildRequires:  pkgconfig(librhsm)
 BuildRequires:  pkgconfig(sqlite3)
 BuildRequires:  pkgconfig(json-c)
 BuildRequires:  pkgconfig(cppunit)
-BuildRequires:  pkgconfig(modulemd) >= 1.4.0
+BuildRequires:  pkgconfig(modulemd) >= %{libmodulemd_version}
 
+Requires:       libmodulemd >= %{libmodulemd_version}
```

Comment:
this is wrong, it should be `libmodulemd%{?_isa} >= …`

Reply:
@ignatenkobrain Please can you explain why?